src/test/mod.rs

```diff
@@ -1,36 +1,34 @@
 pub mod jaegar;
 
-use hyper::{body::Bytes, header::HeaderValue, Body, Error, HeaderMap, Response};
+use http_body_util::BodyExt;
+use hyper::{header::HeaderValue, Error, HeaderMap, Response};
 pub use opentelemetry_api::trace::{SpanId, TraceId};
 use rand::Rng;
 
 #[derive(Debug)]
 pub struct TracedResponse {
-    resp: Response<Body>,
+    resp: Response<hyper::body::Incoming>,
     pub trace_id: TraceId,
     pub span_id: SpanId,
 }
 
 impl TracedResponse {
-    pub fn new(resp: Response<Body>, traceparent: Traceparent) -> Self {
+    pub fn new(resp: Response<hyper::body::Incoming>, traceparent: Traceparent) -> Self {
         Self {
             resp,
             trace_id: traceparent.trace_id,
             span_id: traceparent.span_id,
         }
     }
 
-    pub async fn get_bytes(&mut self) -> Result<Bytes, Error> {
-        hyper::body::to_bytes(self.body_mut()).await
-    }
-
-    pub async fn into_bytes(self) -> Result<Bytes, Error> {
-        hyper::body::to_bytes(self.resp).await
+    #[cfg(feature = "axum")]
+    pub async fn into_axum_bytes(self) -> Result<axum::body::Bytes, Error> {
```

Comment:
`Result<_, hyper::Error>` is the same as `hyper::Result<_>`

Reply:
but we return axum::body::Bytes, is the type the same? (haven't checked yet)

Reply:
We return `Result< axum::body::Bytes , hyper::Error>` and hyper, as most crates, provides custom `Error` together with a custom `Result` type:

```rust
pub type Result<T> = std::result::Result<T, Error>;
```

There is no difference which one to use, but `Result` seems more convenient to me.

Reply:
And it seems like shadowing std `Error` by using `hyper::Error` made this definition a little bit confusing 😃

Reply:
@raphamorim I changed this line after all, hope it's OK:
https://github.com/nentgroup/telemetry-rust/pull/5/commits/456fb3c062e58bf1ffd415195cfa304f9a8afcb2